ghufran/maartjes-work.js

```diff
@@ -0,0 +1,78 @@
+'use strict';
+
+const monday = [
+  {
+    name: 'Write a summary HTML/CSS',
+    duration: 180,
+  },
+  {
+    name: 'Some web development',
+    duration: 120,
+  },
+  {
+    name: 'Fix homework for class10',
+    duration: 20,
+  },
+  {
+    name: 'Talk to a lot of people',
+    duration: 200,
+  },
+];
+
+const tuesday = [
+  {
+    name: 'Keep writing summary',
+    duration: 240,
+  },
+  {
+    name: 'Some more web development',
+    duration: 180,
+  },
+  {
+    name: 'Staring out the window',
+    duration: 10,
+  },
+  {
+    name: 'Talk to a lot of people',
+    duration: 200,
+  },
+  {
+    name: 'Look at application assignments new students',
+    duration: 40,
+  },
+];
+/* cSpell: disable */
+const maartjesTasks = monday.concat(tuesday);
+const maartjesHourlyRate = 20;
+
+function computeEarnings(tasks, hourlyRate) {
+  const duration = tasks.map(task => {
+    // turn mins to hours
+    const hours = task.duration / 60;
+    return { ...task, duration: hours };
+  });
```

Comment:
This code works fine but can be written more efficiently. The `map()` function by default will return a new array with each element being the result of the callback function.

`const duration = tasks.map(task => task.duration / 60);`

Take a look at the docs for some more info on `map()`, https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map